templates/static/js/src/demobox/components/LiveDemoBox/LiveDemoBox.tsx

```diff
@@ -135,6 +136,19 @@ const LiveDemoBox = () => {
               />
             </div>
           </div>
+          <div className="row">
+            <CodepenPrefill
+              label="Edit on CodePen"
+              className="p-button--link p-codepen-link"
+              target="_blank"
+              title="Vanilla framework example"
+              css_external="https://assets.ubuntu.com/v1/vanilla-framework-version-3.1.0.min.css;https://assets.ubuntu.com/v1/4653d9ba-example.css"
+              html={
+                "<div class='p-notification--information'>  <div class='p-notification__content'>    <h5 class='p-notification__title'>Permissions changed</h5>    <p class='p-notification__message'>Anyone with access can view your invited users.</p>  </div></div>"
```

Comment:
Once https://github.com/canonical-web-and-design/vanilla-framework/pull/4321 is merged, the `useCode()` hook can be used to obtain HTML, JS and CSS.